tests/database_test_utils.py

```diff
@@ -0,0 +1,113 @@
+from typing import Any
+
+from sqlalchemy import text
+from sqlmodel import Session, SQLModel
+from tad.core.db import get_engine
+
+
+class DatabaseTestUtils:
+    """
+    Class to use for testing database calls. On creation, this class destroys and recreates the database tables.
+    """
+
+    def __init__(self):
+        self.clear()
+
+    def clear(self) -> None:
+        """
+        Drops and recreates the database tables.
+        :return: None
+        """
+        SQLModel.metadata.drop_all(get_engine())
+        SQLModel.metadata.create_all(get_engine())
+
+    def _enrich_with_default_values(self, specification: dict[str, str | int]) -> dict[str, str | int]:
+        """
+        If a known table dictionary is given, like a task or status, default values will be added
+        and an enriched dictionary is returned.
+        :param specification: the dictionary to be enriched
+        :return: an enriched dictionary
+        """
+        default_specification: dict[str, str | int] = {}
+        if specification["table"] == "task":
```

Comment:
i would recommend making a contants file where you create default test objects (like a task object). This helps with initialization and with validating

Reply:
Good idea, that may work better than creating new objects. For now, I am quite content with the dynamic option of the current solution, maybe in the future we can combine this.